packages/jest-environment-jsdom/src/index.ts

```diff
@@ -15,11 +15,12 @@ import {JSDOM, VirtualConsole} from 'jsdom';
 
 // The `Window` interface does not have an `Error.stackTraceLimit` property, but
 // `JSDOMEnvironment` assumes it is there.
-interface Win extends Window {
-  Error: {
-    stackTraceLimit: number;
+type Win = Window &
+  Global.Global & {
+    Error: {
+      stackTraceLimit: number;
+    };
   };
-}
 
 function isWin(globals: Win | Global.Global): globals is Win {
```

Comment:
If it's always a `Window`, we don't need this guard anymore?

Reply:
Ok, I will update it now!